src/components/flow/routers/subflow/__snapshots__/SubflowRouterForm.test.tsx.snap

```diff
@@ -21,6 +21,7 @@ Array [
         Object {
           "flow": Object {
             "name": "My Subflow Flow",
+            "type": undefined,
```

Comment:
this also feels like a hint that something with my testing approach is off, as here i think the type should either be "messaging" or "voice" and not "undefined"

Reply:
actually, i think this one should also be flowContext.ts -> AssetType.Flow enum -> "flow" (and not "undefined") per https://github.com/nyaruka/floweditor/blob/main/src/components/flow/routers/subflow/SubflowRouterForm.test.tsx#L46, so i'm not sure why that's undefined bc i don't think anything changed with that specific mocked object